utils/installer/config.example-no-ts.lua

```diff
@@ -76,6 +76,37 @@ lvim.builtin.treesitter.highlight.enabled = true
 --     args = {}
 --   }
 -- }
+-- you can define multiple providers for one language
```

Comment:
`you can define multiple language servers`

Reply:
changed the wording here and in another example file